src/custom-types.json

```diff
@@ -637,6 +637,14 @@
   "image/x-xcf": {
     "compressible": false
   },
+  "image/dds" : {
+    "sources": [
+      "https://docs.microsoft.com/en-us/windows/win32/wic/dds-format-overview",
```

Comment:
I cannot find any reference to `image/dds` on this page. Perhaps I'm missing it, though. I just see `image/vnd-ms.dds`. Can you help me find it?

Reply:
actually I might have been mistaken, apparently the MIME type really is `image/vnd.ms-dds`, and not just image/dds, let me fix that.

Reply:
I also confirmed that on this page here: https://www.aconvert.com/image/jpg-to-dds/